src/extensions/manifest.ts

```diff
@@ -56,11 +56,29 @@ export async function writeToManifest(
   await writeEnvFiles(specs, config, options.force);
 }
 
+/**
+ * Remove an instance from extensions manifest.
+ */
+export function removeFromManifest(instanceId: string, config: Config) {
+  if (!instanceExists(instanceId, config)) {
+    throw new FirebaseError(`Extension instance ${instanceId} not found in firebase.json.`);
+  }
+
+  const extensions = config.get("extensions", {});
+  extensions[instanceId] = undefined;
+  config.set("extensions", extensions);
+  config.writeProjectFile("firebase.json", config.src);
+  logger.info(`Removed extension instance ${instanceId} from firebase.json`);
+
+  config.deleteProjectFile(`extensions/${instanceId}.env`);
```

Comment:
Do we want to clean up other .env files like ${instanceId}.env.local or ${instanceId}.env.${project}?

Reply:
Added one line for removing .local since that's easy.

There could be multiple project specific env files, so might take some work. Since they have to be added manually today, and we log explicitly what we removed, we can leave this as a lower priority todo for now.